examples/dpo_demo_gemma3.ipynb

```diff
@@ -0,0 +1,849 @@
+{
+ "cells": [
+  {
```

Comment:
Do you want to give a brief description at the top, as to what the purpose of the example is, and what hardware you are running it on? Might be useful for the user! :)